system/systemd/kazoo-app.service

```diff
@@ -0,0 +1,25 @@
+[Unit]
+Description=Kazoo whistle_app server daemon
+Wants=epmd.socket
+After=syslog.target network.target auditd.service epmd.socket
+
+[Service]
+Type=forking
+User=kazoo
+Group=daemon
+WorkingDirectory=/opt/kazoo
+Environment="ERL_LIBS=/opt/kazoo/deps:/opt/kazoo/core:/opt/kazoo/applications:$(echo /opt/kazoo/deps/rabbitmq_erlang_client-*/deps)"
```

Comment:
This mean - requred to delete string "$(echo /opt/kazoo/deps/rabbitmq_erlang_client-*/deps)"?